### DISCUSSION TITLE
Personal blog on PubPub

### DISCUSSION THREAD

**Discussion by gabestein** [knowledgefutures/pubpub#1433] [Show and tell]
`<shamelessplug>`
I just started using PubPub for my [personal blog](https://gabestein.com),including migrating a bunch of old scattered archives.
`</shamelessplug>`

There are definitely some shortcomings, mostly related to creating tags being too difficult and default page layouts being a bit funky. But overall, I'm pretty pleased with it. The connections feature is particularly fun in the blog context, and otherwise it basically just works.

I wondered if anyone else had tried similar and had thoughts/feedback/ideas.

**Comment by jeffpooley** [comment]
Looks great—subscribed!

**Reply by writingpoli** [reply]
I tried something similar a while back as a subunit of SAR, but didn't like that it would push out my personal posts on the same RSS feed (a toggle to exclude from feeds might be a good idea to add to the roadmap). 

o I deleted it and created a new community,  but I haven't imported anything yet: https://inadequate.pubpub.org/

**Reply by gabestein** [reply]
Cool! Let me know if you ever get up to anything with it! I think we're pretty close to being a decent solution for it, and have begun logging a few enhancement requests to cover the use-case.
